lib/msf/core/exploit/file_dropper.rb

```diff
@@ -250,6 +254,26 @@ def file_dropper_deleted?(session, path, exists_before)
     end
   end
 
+  # Check if the path being removed is the same as the working directory
+  #
+  # @param [String] path The path to check
+  # @return [Boolean] true if the path is the same, otherwise false
+  def file_dropper_check_cwd?(path)
+    if session.type == 'meterpreter'
+      return true if path == session.fs.dir.pwd
+    else
+      pwd = if session.platform == 'windows'
```

Comment:
This syntax is highly unreadable 😞

Reply:
I don't think a ternary would be much better. But I agree. I was trying for https://github.com/bbatsov/ruby-style-guide#use-if-case-returns. Let's split the assignment across two lines.